test/git-resolver/gitea.yaml

```diff
@@ -0,0 +1,853 @@
+# GENERATED BY RUNNING `helm template -n default gitea gitea-charts/gitea > gitea.yaml`
```

Comment:
what's this file for?

Reply:
Spinning up gitea in the cluster for the git API resolution e2e tests. I'll add a comment.